### DISCUSSION TITLE
How to make uosc show .m3u in a it's own file manager?

### DISCUSSION THREAD

**Discussion by pinksquare** [tomasklaen/uosc#926] [Q&A] [answered]
uosc can open .m3u files . For example: if  i add the whole folder with a .m3u file in it to the playlist using ALT+mouse click - playlist file loads as expected . But navigating right to the same folder with inner file manager - doesn't see anything .

**Comment by hooke007** [comment]
https://github.com/tomasklaen/uosc/blob/1c38d4d3a7405f1aac603ba02f846c3d84dac96b/src/uosc.conf#L191

**Comment by christoph-heinrich** [comment, marked as answer]
Your `uosc.conf` has an option `video_types=3g2,3gp,asf,avi,...` (and similar for audio and images). Add the playlist extension to one of those three and it will show up in the file manager. It's not a nice solution, as that will also cause playlists to show up in menus that really shouldn't show playlists, but it's a workaround for now.

**Reply by pinksquare** [reply]
Yeah , it works. Thanks .

**Reply by christoph-heinrich** [reply]
FYI the next release has a `playlist_types` option.
